Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1

```diff
@@ -234,11 +234,19 @@ function Main {
             -RunInBackground -runAsSudo
 
         Write-LogInfo "Monitoring test run..."
+        $ForceExitCount = 0
+        $MaxCount = 10
         while ((Get-Job -Id $testJob).State -eq "Running") {
             $currentStatus = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
```

Comment:
Add this code here : 
```
if ($currentStatus -inotmatch "Doing forceful exit of this job") {
    $FioStuckCounter = 0
}
```

Reply:
added else condition to reset the counter